templates/cloudflare-workers/remix.config.js

```diff
@@ -1,6 +1,5 @@
 /** @type {import('@remix-run/dev').AppConfig} */
-module.exports = {
-  devServerBroadcastDelay: 1000,
+export default {
   ignoredRouteFiles: ["**/.*"],
   server: "./server.ts",
   serverConditions: ["worker"],
```

Comment:
I think this should be `["workerd", "worker", "browser"]`:

- `workerd` is the new standard for Cloudflare Workers and could have slightly different code, usually optimised for edge runtimes. This is being adopted by React [with a different bundle from their browser bundle](https://github.com/facebook/react/blob/559e83aebb2026035d47aa0ebf842f78d4cd6757/packages/react-dom/package.json#L52-L58).
- `worker` should remain, because it’ll usually work and rely on less code than the browser bundle.
- `browser` lets us fall back to browser code in packages that don’t have one of the other builds. Anecdotally, some of my code breaks without adding this condition. I’d argue this is more likely to happen than the inverse (a user imports a code that relies on browser-specific globals that Workers doesn’t have).